bears/vcs/git/GitCommitBear.py

```diff
@@ -25,6 +25,16 @@ class GitCommitBear(GlobalBear):
     LICENSE = 'AGPL-3.0'
     ASCIINEMA_URL = 'https://asciinema.org/a/e146c9739ojhr8396wedsvf0d'
     CAN_DETECT = {'Formatting'}
+    ISSUE_INFO = {
+        'github': {
+                    'issue': r'(?:\w+/\w+)?#(\S+)',
+                    'full issue': r'https?://github\S+/issues/(\S+)'
+                  },
+        'gitlab': {
+                    'issue': r'(?:\w+/\w+)?#(\S+)',
+                    'full issue': r'https?://gitlab\S+/issues/(\S+)'
+                  }
```

Comment:
add trailing commas (everywhere)

Reply:
Should I put them in the SUPPORTED_HOST_KEYWORD_REGEX as well ?
(or that be left for a newcomer issue ?)